### PR TITLE
Added version warning

### PR DESCRIPTION
**Pull request by mikeWShef**
Version warning as discussed in issue: PYMC 2 warning #196

### CODE REVIEW

File: docs/INSTALL.rst [hunk 1]
```diff
@@ -10,6 +10,8 @@ Installation
 :License: PyMC is released under the Academic Free license.
 :Version: 2.3.7
 
+**NOTE: The current version PyMC (version 3) has been moved to its own repository called pymc3. Unless you have a good reason for using this package, we recommend all new users adopt PyMC3.**
```
Comment (twiecki) [inline docs/INSTALL.rst]: can you add a link?

**Comment by twiecki** [comment]
Thanks @mikeWShef! @fonnesbeck can you recompile the docs when you get a chance?

**Comment by mikeWShef** [comment]
I've just looked at the page and it looks like links don't work if they are bold (at least on github), 13 needs to be changed to:

**NOTE:** The current version PyMC (version 3) has been moved to its own repository called `pymc3`_. Unless you have a good reason for using this package, we recommend all new users adopt PyMC3.

**Comment by twiecki** [comment]
@mikeWShef that would require a new PR, can you make one?

**Comment by fonnesbeck** [comment]
Docs are recompiled. You'll need to clear your browser cache to see the update.

**Comment by twiecki** [comment]
@fonnesbeck I also put out a new version on pypi.

**Pull request merged**